nilearn/datasets/atlas.py

```diff
@@ -576,3 +576,73 @@ def fetch_yeo_2011_atlas(data_dir=None, url=None, resume=True, verbose=1):
         url=url,
         resume=resume,
         verbose=verbose)
+
+
+def fetch_atlas_aal(data_dir=None, url=None, resume=True, verbose=1):
+    """Download and return file names for the AAL for SPM 12 parcellation.
+
+    The provided images are based on the spatially normalized single-subject
+    high-resolution T1 volume provided by the Montreal Neurological Institute
+    (MNI)
+
+    Parameters
+    ----------
+    data_dir: string
+        directory where data should be downloaded and unpacked.
+
+    url: string
+        url of file to download.
+
+    resume: bool
+        whether to resumed download of a partly-downloaded file.
+
+    verbose: int
+        verbosity level (0 means no message).
+
+    Returns
+    -------
+    data: sklearn.datasets.base.Bunch
+        dictionary-like object, keys are:
+
+        - "anat": anatomy image.
+
+        - "labels": labels id in an xml file
+
+    Notes
+    -----
+    For more information on this dataset's structure, see
+    http://www.gin.cnrs.fr/AAL-217?lang=en
+
+    N. Tzourio-Mazoyer, B. Landeau, D. Papathanassiou, F. Crivello,
+    O. Etard, N. Delcroix, B. Mazoyer, and M. Joliot.
+    NeuroImage 2002. 15 :273-28
+
+    Licence: unknown.
+    """
+    spm_version = 12
```

Comment:
yes they are quite different, i downloaded all the versions to test, spm12 is the first to have only a nii file and xml file, all the previous ones have a crazy amount of files, i think they have one file per label.